apps/backend/src/modules/projects/project.service.ts

```diff
@@ -135,10 +151,20 @@ class ProjectService implements Service {
 		};
 	}
 
-	public async findAllWithoutPagination(): Promise<
-		ProjectGetAllItemResponseDto[]
-	> {
-		const projects = await this.projectRepository.findAllWithoutPagination();
+	public async findAllWithoutPagination({
+		hasRootPermission,
+		userProjectIds,
```

Comment:
let's pass userProjectIds optionally, depending on whether user has root access. We only need to pass it when we want to check userProjectIds
```suggestion
		userProjectIds,
```